javascript/javascript1/week4/VoiceAssistant.js

```diff
@@ -0,0 +1,180 @@
+const commandHistory = []
+let curUser = ""
+const todos = []
+
+function getReply(command)
+{
+// save the name and respond with a message
+    //Checking whether the user already indtroduced 
+    let found = false;
+    for(item in commandHistory) 
+    {
+        if(commandHistory[item] === command.toLowerCase()) 
+        {
+            found = true;
+            break;
+        }
+    }
```

Comment:
It doesn't look like you need the `commandHistory` array.
You're using it for the same thing as the `currentUserName`.